src/components/webviews/jsInteractions/jsCozyInjection.js

```diff
@@ -19,13 +19,18 @@ const makeMetadata = routeName => {
   })
 }
 
-export const jsCozyGlobal = (routeName, isSecureProtocol) => `
+export const jsCozyGlobal = (
+  routeName,
+  isSecureProtocol,
+  isNativePassInstalled
+) => `
   if (!window.cozy) window.cozy = {}
   window.cozy.isFlagshipApp = true
   window.cozy.ClientKonnectorLauncher = 'react-native'
   window.cozy.ClientConnectorLauncher = 'react-native' // deprecated
   window.cozy.flagship = ${makeMetadata(routeName)}
   window.cozy.isSecureProtocol = ${isSecureProtocol || 'false'}
+  window.cozy.isNativePassInstalled = ${isNativePassInstalled || 'false'}
```

Comment:
I would rename it `isNativePassInstalledOnTheDevice` since it can be confusing with "at least branched once on the cozy".

Reply:
Also, I don't like to give this information to all the apps. Also I think we should avoid relying on window.cozy stuff for this kind of information.

Wondering if the cozy-pass-web should not call an intent (webapp -> native -> webapp) like : `webview.call(isNativePassInstalledOnDevice)`

Reply:
I replace it with an intent with `isNativePassInstalledOnDevice`.